### PR TITLE
added Server Listener Interface methods

### PR DESCRIPTION
**Pull request by jeinglis**

### CODE REVIEW

File: AgniMessenger/src/agni/server/receiver/ChatListener.java [hunk 1]
```diff
@@ -2,4 +2,5 @@
 
 public interface ChatListener {
 
+    public void chatRequest (ip, message);
```
Comment (ExtraGravity) [inline AgniMessenger/src/agni/server/receiver/ChatListener.java]: Don't these need types? The types would be `InetAddress` and `String`.

File: AgniMessenger/src/agni/server/receiver/ChatListener.java [hunk 2]
```diff
@@ -2,4 +2,9 @@
 
 public interface ChatListener {
 
+	/*
+	 * Receives user requests from the ChatReceiver
+	 * ip will have been converted from inet to String
+	 */
```
Comment (ExtraGravity) [inline AgniMessenger/src/agni/server/receiver/ChatListener.java]: A bunch of tabs in this PR, example here should be changed to spaces.

File: AgniMessenger/src/agni/client/communication/MessageReceiver.java [hunk 3]
```diff
@@ -1,6 +1,10 @@
 package agni.client.communication;
 
 public class MessageReceiver implements Runnable {
+	
+	public void MessageReceiver(){
+		
+	}
```
Comment (ExtraGravity) [inline AgniMessenger/src/agni/client/communication/MessageReceiver.java:7]: Why is this file changed in this PR?

File: .project [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+<projectDescription>
+	<name>AgniMessenger</name>
+	<comment></comment>
+	<projects>
+	</projects>
+	<buildSpec>
+	</buildSpec>
+	<natures>
+	</natures>
+</projectDescription>
```
Comment (ExtraGravity) [inline .project:11]: Your .project is still in a funky spot.

File: AgniMessenger/src/agni/client/communication/MessageReceiver.java (hunk 3 above)
Comment (jeinglis) [inline AgniMessenger/src/agni/client/communication/MessageReceiver.java:7]: because I was going to make a change and realized I was on the wrong branch
